charts/timber-fluentd/values.yaml

```diff
@@ -0,0 +1,124 @@
+global:
+  # -- Extra pod labels in a map[string]string format, most likely to be used for the costing labels.
+  extraPodLabels: {}
+
+nameOverride: ""
+
+image:
+  # -- Docker registry for fluentd image
+  registry: ghcr.io
+  # -- Docker image repository for fluentd
+  repository: caraml-dev/timber/fluentd
+  # -- Docker image tag for fluentd
+  tag: v0.0.0-build.16-01ac82e
+  # -- Docker image pull policy
+  pullPolicy: IfNotPresent
+
+replicaCount: 1
+
+# -- Resources requests and limits for fluentd StatefulSet. This should be set
+# according to your cluster capacity and service level objectives.
+# Reference: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+resources: {}
+
+# GCP configurations for flushing Observation logs to BQ
+gcpServiceAccount:
+  # -- Flag to toggle flushing Observation logs to BQ
+  enabled: false
+  # -- Base64 encoded service account json
+  account: ""
+
+# -- PVC configurations for fluentd StatefulSet storage
+pvcConfig:
+  name: cache-volume
+  mountPath: /cache
+  storage: 3Gi
+
+# -- Kubernetes Service for fluentd StatefulSet
+service:
+  type: ClusterIP
+  internalPort: &internalPort 9880
+  externalPort: &externalPort 24224
+  multiPort:
+    enabled: true
+  multiPorts:
```

Comment:
Should this whole block become a part of the template? How much flexibility may we need here?

Reply:
hmm thats a good point, I dont think its required at this point